### PR TITLE
allow elution buffer for fastq

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR adds a fix for the missing elution buffers from Orderportal

**How to prepare for test**:
- [x] test through [cgweb](https://github.com/Clinical-Genomics/cgweb/pull/174)

**Review:**
- [x] code approved by CR
- [x] tests executed by @patrikgrenfeldt 
- [x] "Merge and deploy" approved by @patrikgrenfeldt 
Thanks for filling in who performed the code review and the test!

This is patch **version bump** because it fixes a non-concordance with the current orderform

### CODE REVIEW

File: cg/meta/orders/schema.py [hunk 1]
```diff
@@ -203,7 +203,7 @@ def validate(self, *value):
 }
 
 FASTQ_SAMPLE = {
-    # Orderform 1508:18
+    # Orderform 1508:?
```
Comment (northwestwitch) [inline cg/meta/orders/schema.py]: What stands this ? for ?

Comment (patrikgrenfeldt) [reply cg/meta/orders/schema.py]: it was following # Orderform 1508:18 before, now I have made changes and I haven't verified against any particular orderform.. thoughts on what to do?

**Comment by patrikgrenfeldt** [comment]
deployed

File: cg/meta/orders/schema.py (hunk 1 above)
Comment (northwestwitch) [reply cg/meta/orders/schema.py]: naa, it's fine!

**Pull request merged**